store/mockstore/unistore/tikv/mvcc.go

```diff
@@ -935,16 +935,16 @@ func (store *MVCCStore) buildPrewriteLock(reqCtx *requestCtx, m *kvrpcpb.Mutatio
 		lock.Op = uint8(kvrpcpb.Op_Put)
 	}
 	if rowcodec.IsRowKey(m.Key) && lock.Op == uint8(kvrpcpb.Op_Put) {
-		if rowcodec.IsNewFormat(m.Value) {
```

Comment:
When `rowcodec.IsNewFormat(m.Value)` , the previous code are deleted?

Reply:
Here it sets `reqCtx.buf = m.Value` and later `lock.Value = reqCtx.buf`. The main purpose is to set `lock.Value = m.Value`. In https://github.com/pingcap/tidb/pull/40422/files/f2fe4bde5508cad89d582145709fa286a24bec18#diff-c8058e505bb9be555aca4f1a51fbf20e712ea4d6d89812a09f3ebf02ff99e2f2R896, the `lock.Value` has been initialized with `m.Value`, so there is no need to assign again.